app/src/display/widgets/Kconfig

```diff
@@ -3,6 +3,8 @@
 
 menu "ZMK Display Widgets"
 
+if ZMK_DISPLAY_STATUS_SCREEN_BUILT_IN
+
```

Comment:
Instead of this, I think we should change the defaults or each widget so they're not defaulted on, and instead `imply` them all or the build in screen choice. That feels aligned with the same idea of "higher level aggregating in code *and* config" like for your custom screen.

Reply:
I removed the defaults and implied then all ***on*** the built-in screen (that or was confusing me 😅). It seems the `if` of the `default y if...` was unnecessary given the `depend on` lines were already there. I can build with or without the built-in screen option and they work properly on the left and right sides of my corne!